### ISSUE TITLE
[TF2] Feature Request: Custom particle manifests

### ISSUE THREAD

**Issue by Blunderstab** [ValveSoftware/Source-1-Games#4505]
Being able to send players a temporary custom particle manifest file when they're joining a server would be an absolute godsend, _especially_ for custom weapon servers. There have been so many times when I've dug through existing particles to try and find something to convey information to players and there's just nothing that fits the bill. Considering how simple it is to use custom models and sounds, it seems like a strange exclusion to not have something for custom particles.

Thank you so much for continuing to update this game.

**Comment by Lizard-Of-Oz** [comment]
I agree that this is important. Currently, you can't load/precache a particle file (or a sound manifest) without baking them into the map, which isn't always an option for custom gamemodes on community servers.
